### PR TITLE
Add missing readtime

### PR DESCRIPTION
**Pull request by wu-hui**

### CODE REVIEW

**Comment by wu-hui** [comment]
@dconeybe This is same as https://github.com/firebase/firebase-ios-sdk/pull/10946 

I added a test, but somehow force pushed then closed the PR, I cannot reopen that one apparently.

File: Firestore/core/test/unit/remote/serializer_test.cc [hunk 1]
```diff
@@ -1124,6 +1124,42 @@ TEST_F(SerializerTest, EncodesFirstLevelKeyQueries) {
...
+                                           model::SnapshotVersion::None()),
+                    proto);
+  }
+
+  {
+    SCOPED_TRACE("EncodesTargetDataWithResumeByReadTime");
+    v1::Target proto;
+    proto.mutable_documents()->add_documents(ResourceName("docs/1"));
+    proto.set_target_id(1);
+    proto.mutable_read_time()->set_seconds(1000);
```
Comment (dconeybe) [inline Firestore/core/test/unit/remote/serializer_test.cc:1154]: nit: Use a different value for `seconds` and `nanos` to make sure that the round trip doesn't accidentally invert them.

**Review by dconeybe — APPROVED** [review]
Thanks for adding the test! LGTM. Just one minor comment about a small improvement to the test.

File: Firestore/core/test/unit/remote/serializer_test.cc (hunk 1 above)
Comment (wu-hui) [reply Firestore/core/test/unit/remote/serializer_test.cc:1154]: Done.

**Pull request merged**